shared/actions/platform-specific.js.flow

```diff
@@ -14,6 +15,35 @@ type NextURI = string
 declare function saveAttachmentDialog(filePath: string): Promise<NextURI>
 declare function displayNewMessageNotification(text: string, convID: string, badgeCount: number): void
 
+// RouteStateStorage is a platform-specific implementation of route
+// state storage, e.g. storing the current loaded tab so that it is
+// restored on the next load.
+//
+// Usage:
+//
+//   const routeStateStorage = new RouteStateStorage()
+//
+//   // Once on app load:
+//   await dispatch(routeStateStorage.load)
+//   // Read e.g. config.initialTab from the state and navigate to it.
+//
+//   // On a tab switch:
+//   dispatch(routeStateStorage.store)
+//
+// Implementations should ignore stores that happen when
+// getState().routeTree.routeChanged is false, since that is set when
+// a user has performed a navigation action, or the state from .load
+// has been applied, and we don't want to store any state until either
+// of those two events happens.
+declare class RouteStateStorage {
```

Comment:
I used to have to store shared state between load and store, but don't anymore, so I might just remove this class.